.muse.toml

```diff
@@ -0,0 +1 @@
+jdk11 = true
```

Comment:
Please remove this from the PR

Reply:
Woops, thanks for the catch that was for my own use.